notebook/converted/simulation_results.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import collections
+import glob
+import json
+import os
+
+import numpy as np
+import pandas as pd
+from plotnine import *
+
+from saged import utils
+
+
+# ## Set Up Functions and Get Metadata
+
+# In[2]:
+
+
+def split_sample_names(df_row):
+    train_samples = df_row['train samples'].split(',')
+    val_samples = df_row['val samples'].split(',')
+    
+    return train_samples, val_samples
+
+def create_dataset_stat_df(metrics_df, sample_to_study, 
```

Comment:
Could you put any of these functions into a file that's shared between analysis scripts? This code looks similar to what you have in some of your other notebooks (although maybe I'm missing some differences).

Reply:
Yeah, that's a good idea. I copied the logic over to generate results quickly, but it definitely needs DRYing